scripts/config-dhcp.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+#copy the configuration information to network-conf.txt
+echo "config-server" > /home/pi/network-conf
+
+#Bring down interface eth0
+sudo ifdown eth0
+echo "Interface eth0 is down"
+
+#Bring up eth0 with DHCP Server configuration
+sudo ifup eth0
+echo "Interface eth0 is up with DHCP Server configuration"
+
+#Add configuration settings for dhcp-server to dhcp/dhcpd.conf
```

Comment:
The proceeding can also be replaced by just committing the .conf and overwriting the system file (once) during update. I'm glad you've had some practice with `sed`, though. ;)